sdk/Sdk/Extensions/CustomAttributeExtensions.cs

```diff
@@ -4,7 +4,6 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
 using Mono.Cecil;
 
 namespace Microsoft.Azure.Functions.Worker.Sdk
```

Comment:
Moved 2 existing extension method classes under the new "Extensions" directory.They were in the root of the project earlier.